docs/development/pigs/pig-024.rst

```diff
@@ -0,0 +1,277 @@
+release).
+
+.. note::
+    As the Gammapy community fully supports Open Science, we strongly encourage authors using Gammapy to follow
+    the FAIR4RS principles and to allow the reproducibility of their results. As consequence, we suggest always mentioning
+    the `Zenodo <https://zenodo.org/>`_ DOI or the `HAL <https://hal.archives-ouvertes.fr/>`_ SWHID identifier (associated with
+    the `Software Heritage <https://www.softwareheritage.org/>`_ archive) of the used release of Gammapy.
+
+This citation scheme with these **two references** will be given on our web pages.
+
+3. Authorship policy
+====================
+
+The Gammapy references contain a list of authors that requires to be updated with time according to a
+general policy. This section defines what is a `contributor` to Gammapy and the policy for each type of product.
+
+Definition of a `Contributor`
+-----------------------------
+
+Contributing to Gammapy can be made in different ways:
+
+- contributing to gammapy source code,
+- contributing to our documentation (rst pages, docstring, gammapy-webpage),
+- contributing to our DevOps environment (e.g. CI, docs and package deployment, GitHub configuration),
+- contributing to our associated projects (gammapy-benchmarks, gammapy-data, gammapy-extra),
+- contributing to the maintenance of Gammapy products on open-source repositories (conda-forge, PyPI, Zenodo, HAL, docker-hub, etc) or open hub (openhub, etc),
+- leading the precise tasks identified by the Lead Developers,
+- organizing and leading hands-on sessions or schools,
+- managing our social media accounts,
+- and coordinating the project by the Lead Developers.
+
+Any contribution without threshold on these topics renders a user as official Gammapy contributor. And the contributions
+are tracked using the GitHub contributions. However, the use of personal data is regulated in Europe and one should have the
+authorisation of users to be cited with their personal information (typically, their affiliation and their ORCID if any).
+For this reason, we propose to use the GitHub `Developer Certificate of Origin <https://github.com/apps/dco>`_ (DCO) on
+Pull Request. Its acceptation by contributors will permit to:
+
+- certify that a user wrote or otherwise has the right to submit a code under our open source licence,
+- allow our project to use their personal data for the contributions' record (ie for releases).
+
+The file ``README.md`` will contain a `Contributing` section explaining that the project follows the GitHub DCO, as well
+as a link to this PIG.
+
+.. important::
+    We strongly recommend that the contributors use always the same GitHub account with a valid full name, to
+    synchronise their git email address with their GitHub address, to have a unique `ORCID <https://orcid.org/>`_
+    identifier and store it into their GitHub user profile for commodity, and inform the developers of any change of
```

Comment:
As discussed in Slack and more details below, we should not rely on people's GitHub user profiles.

Rather we should ask people to contribute the information they want to have listed for their Gammapy contributions here, which also would directly provide the explicit consent necessary for the usage of the data

Reply:
We are not relying on them, but we will use them. A call is made to authors, as described in the sections of releases, papers and conferences

Reply:
I think there is a misunderstanding. We will not use GitHub profile data in an automatized way, but it can be a way to reach out to people to collect the data for the first `ALL_AUTHORS_EVER.yaml` file. So we will send around a request, containing a link to a Google form or request to open a PR to everyone who is in the git history of Gamapy up to now. In individual cases we might look at their GitHub profiles to get e.g. an update to date mail address to send the request to. 
Once this first `ALL_AUTHORS_EVER.yaml` is theere (hopefully complete based on voluntary filling of the Google form...) we will just maintain it by hand. New contributors will add their information or once an affiliation changes people need to submit a PR to update it.

The only controversial thing to me is, how to get this first `ALL_AUTHORS_EVER.yaml` as complete as possible. In this case we might announce to use the name of the people (no affiliation, no ORCID, no mail-address) if they do not object. In future this will happen via the DCO.